site/content/how-to/traffic-management/advanced-routing.md

```diff
@@ -59,10 +56,9 @@ spec:
 EOF
 ```
 
-This Gateway defines a single listener on port 80. Since no hostname is specified, this listener matches on all hostnames.
+This gateway defines a single listener on port 80. Since no hostname is specified, this listener matches on all hostnames.
```

Comment:
So far, all of the Gateway API CRD names -- HTTPRoute, Gateway, GatewayClass, and ReferenceGrant -- have been capitalized. Can we stick to that convention?

Reply:
there is a separate conversation about this. I will follow up with you